networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/topics/AttesterSlashingTopicHandlerTest.java

```diff
@@ -55,9 +55,21 @@ public void handleMessage_validSlashing() {
     assertThat(result).isCompletedWithValue(ValidationResult.Valid);
   }
 
+  @Test
+  public void handleMessage_invalidSlashing_wrongFork() {
+    final AttesterSlashing slashing = dataStructureUtil.randomAttesterSlashingAtSlot(wrongForkSlot);
+    when(processor.process(slashing))
+        .thenReturn(SafeFuture.completedFuture(InternalValidationResult.ACCEPT));
```

Comment:
Same as the aggregateTopicHandlerTest, remove this, and verifyNoInteractions on processor...